text/0000-roadmap-2021.md

```diff
@@ -0,0 +1,127 @@
+
+* What is expected of members to be part of this group?
+* Who is the point of contact for questions on the state of this group?
+* Where does this group work?
+* What does this group do?
+* How does this group make decisions?
+
+### What is expected of members to be part of this group?
+
+The membership expectations can be diverse for various groups. Some groups can meet multiple times a week and expect team members to devote time outside these meetings to drive proposals and otherwise contribute, while other teams can have no synchronous meetings and expect bursts of work from time to time.
+
+Especially when multiple teams are interfacing, or a new member is asked to join a team, having this documentation is immensely valuable to communicate up front the ground rules for involvement.
+
+### Who is the point of contact for questions on the state of this group?
+
+This question is aimed at aiding parties outside the team to identify who to approach with questions or feedback. This may be a team lead, but could also just be a point of contact.
+
+Currently, not all teams have such a clearly defined point of contact. This makes it hard for people seeking to interface with the team to establish communications, especially when they don’t currently know Rust’s governance.
+
+### Where does this group work?
+
+This can include the meeting platforms used and chat channels. It’s the contact information for the whole group, not just the contact person noted previously.
+
+### What does this group do?
+
+The goal is to provide a summary of scope for the team. In particular, this is the section that other teams and contributors will read to make decisions for when to consult this team.
+
+This section may also include links to subgroups where that makes sense, for example the compiler and dev-tools teams have a large number of subgroups which should be documented here.
+
+### How does this group make decisions?
+
+This section establishes the process (or processes) by which the team makes decisions. This is useful for both team members and outside parties. Any normative documents produced by the team (for example, the Reference for lang team) should also be listed here.
+
+# Unifying processes and vocabulary
+
+Today, Rust teams decide individually where to host documents and have discussions. We do not want to take away these choices from the Rust teams: we believe that teams know best what works for them. However, sometimes the divergence here is unintentional, and providing defaults that work well can be an excellent way to remove friction.
+
+The specific details of the vocabulary and processes that we will seek to unify are not yet clear, primarily because there is no one place where all of the specifics of team activity are laid out. As we begin to establish charters over the course of the year, we will begin to identify common needs and current choices, and will work to diverge only when needed.
```

Comment:
Can you give an example of where vocabulary diverges today?

Reply:
* Project groups vs. working group
* Major change proposals exist for lang/compiler; not really a thing in other teams though (also a divergence, just different type)

Some of this is not so much about "words" but rather the underlying processes too: a clearer and more uniform structure on how to approach teams with ideas (internals? RFC? MCP? Zulip? Email?) is part of this too. We don't want to impose structure, though, mostly about working to identify where the same structure already exists (or teams want to make use of it) and calling it by the same name so things are more approachable.